### ISSUE TITLE
Missing icons

### ISSUE THREAD

**Issue by davejamesmiller** [joeyespo/grip#85]
This may be the same as #81 - the Octicons don't seem to be loading:

![image](https://cloud.githubusercontent.com/assets/236616/4876089/ec516c8a-62b4-11e4-8484-3015bd8a0925.png)

In the Chrome console log it shows it's failing to load the font:

```
Failed to load resource: the server responded with a status of 404 (NOT FOUND) http://192.168.32.84:5000/grip-cache/octicons/octicons/octicons-192e5fbb927be7cdb8bdfa0ecb98d3574bd395e58f9d00e5dcdfce5bc9660ac0.woff
Failed to load resource: the server responded with a status of 404 (NOT FOUND) http://192.168.32.84:5000/grip-cache/octicons/octicons/octicons-1b8fabf8a76f5d60ef9999b3fcf5e06b9f563148d6215465df8952fdbb9aa3ec.ttf
```

And the same items appear in the SSH console:

```
192.168.32.84 - - [02/Nov/2014 17:24:02] "GET /grip-cache/octicons/octicons/octicons-192e5fbb927be7cdb8bdfa0ecb98d3574bd395e58f9d00e5dcdfce5bc9660ac0.woff HTTP/1.1" 404 -
192.168.32.84 - - [02/Nov/2014 17:24:02] "GET /grip-cache/octicons/octicons/octicons-1b8fabf8a76f5d60ef9999b3fcf5e06b9f563148d6215465df8952fdbb9aa3ec.ttf HTTP/1.1" 404 -
```

I have the latest version:

```bash
$ grip --version
Grip 3.0.0
```

I have tried reinstalling Grip and clearing the cache also:

```bash
$ sudo pip uninstall grip
$ sudo pip install grip
$ grip --clear
```

I've also checked it in Firefox and IE - it's not working there either.

Any ideas?
Thanks

**Comment by abourguignon** [comment]
It affects me as well, a bit sad because it degrades the sleek and neat look'n'feel.

**Comment by joeyespo** [comment]
Fixed with #95.

**Comment by joeyespo** [comment]
FYI, I just released 3.1.0 which fixes this. Be sure to `pip install --upgrade grip`. Thanks for your patience, everyone!

**Comment by davejamesmiller** [comment]
Great, thank you :+1:

**Comment by abourguignon** [comment]
Many thanks !

**Comment by akaleeroy** [comment]
Icons still 404 in 3.3.0

**Comment by joeyespo** [comment]
@akaleeroy Looks like GitHub changed their URL format for assets. Fixed in f04eab8d760a1a6fee6e329e0acbf712af365662. I'll publish a new release soon.

**Comment by joeyespo** [comment]
@akaleeroy FYI, released last week. `pip install --upgrade grip` and that should fix the latest missing icons problem. Thanks for the report!
